packages/x-date-pickers/src/DateCalendar/tests/localization.DateCalendar.test.tsx

```diff
@@ -9,27 +9,34 @@ import 'dayjs/locale/he';
 import 'dayjs/locale/fr';
 import 'moment/locale/he';
 import 'moment/locale/fr';
+import moment from 'moment';
 
 const ADAPTERS_TO_USE: AdapterName[] = ['date-fns', 'dayjs', 'luxon', 'moment'];
 
 describe('<DateCalendar /> - localization', () => {
   ADAPTERS_TO_USE.forEach((adapterName) => {
     describe(`with '${adapterName}'`, () => {
-      const { render } = createPickerRenderer({
-        locale: adapterName === 'date-fns' ? he : { code: 'he' },
-        adapterName,
-      });
+      describe('with wrapper', () => {
```

Comment:
Added this `with wrapper` to clearly show the duality with the `without wrapper` describe below, since I got confused a few times while debugging 😅